torch/csrc/lazy/core/shape.cpp

```diff
@@ -13,6 +13,14 @@ namespace lazy {
 Shape::Shape(at::ScalarType scalar_type, c10::ArrayRef<int64_t> sizes)
     : scalar_type_(scalar_type), sizes_(sizes.begin(), sizes.end()) {}
 
+Shape::Shape(
+    at::ScalarType scalar_type,
+    c10::ArrayRef<int64_t> sizes,
+    c10::optional<std::vector<bool>>& is_symbolic)
```

Comment:
could we make `is_symbolic` a default argument, so we don't need boilerplate for another constructor?